docker-compose.yaml

```
@@ -1,5 +1,53 @@
 
 services:
+
+  gateway:
+    build:
+      context: .
+      dockerfile: services/gateway/Dockerfile
+    ports:
+      - 8083:8080
+
+      
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)